invertedai_cpp/invertedai/drive_request.h

```diff
@@ -30,29 +30,104 @@ class DriveRequest {
 
 public:
   DriveRequest(const std::string &body_str);
+  /**
+   * Serialize all the fields into a string.
+   */
   std::string body_str();
+
+  /**
+   * Update the drive request with the information(agent_states,
+   * agent_attributes, recurrent_states) in the initialize response.
+   */
   void update(const InitializeResponse &init_res);
+  /**
+   * Update the drive request with the information(agent_states,
+   * recurrent_states) in the drive response.
+   */
   void update(const DriveResponse &drive_res);
```

Comment:
Probably one will shadow the other. I assume the first one is not necessary.

Reply:
Not sure if I get your idea, but these two `update` are both used in the client_example, because at the step 0, we don't have drive response, so we use the initialize response to update the drive request.